types/selenium-webdriver/index.d.ts

```diff
@@ -2350,6 +2350,9 @@ export class Builder {
  * @final
  */
 export class By {
+  using: string;
```

Comment:
Please add comments to match the rest of the document. LGTM otherwise.

Reply:
The parameter comments are at lines 2357 and 2358.